test/getGrist.ts

```diff
@@ -198,13 +204,24 @@ export class GristUtils extends GristWebDriverUtils {
     await this.waitForServer();
   }
 
-  public async clickWidgetPane() {
+  // This method doesn't work in single user mode, need to fix upstream.
+  // For now it is  just overridden with a simpler version.
```

Comment:
Can you open a task to do this so it's not completely forgotten? We need to keep the test harness in synch as much as possible between core and widgets.

Reply:
It is overridden, so this is still in sync.